lua/tardis/controls/sh_control_teleport.lua

```diff
@@ -27,7 +27,7 @@ TARDIS:AddControl({
     end,
     serveronly=true,
     power_independent = false,
-    bypass_console_toggle = true,
+    bypass_console_toggle = false,
```

Comment:
Might as well just remove these lines rather than having them to false on this and the double teleport control

Reply:
b70f996f3de524374052efcd1b117246cdb24335